assets/js/modules/analytics-4/components/audience-segmentation/dashboard/AudienceSegmentationSetupCTAWidget.test.js

```diff
@@ -126,6 +126,12 @@ describe( 'AudienceSegmentationSetupCTAWidget', () => {
 			.dispatch( MODULES_ANALYTICS_4 )
 			.finishResolution( 'getReport', [ options ] );
 
+		registry
+			.dispatch( CORE_USER )
+			.finishResolution( 'getPromptDismissCount', [
+				AUDIENCE_SEGMENTATION_SETUP_CTA_NOTIFICATION,
+			] );
+
```

Comment:
As above, `getPromptDismissCount` doesn't have a resolver so it doesn't make sense to try to interact with it.

You should instead populate and resolve `getDismissedPrompts()`.

```suggestion
		const currentTimeInSeconds = Math.floor( Date.now() / 1000 );

		registry.dispatch( CORE_USER ).receiveGetDismissedPrompts( {
			[ AUDIENCE_SEGMENTATION_SETUP_CTA_NOTIFICATION ]: {
				expires: currentTimeInSeconds - 10,
				count: 0,
			},
		} );

		registry
			.dispatch( CORE_USER )
			.finishResolution( 'getDismissedPrompts', [] );
```